src/main/java/frc/robot/subsystems/ControlPanelSubsystem.java

```diff
@@ -0,0 +1,138 @@
+package frc.robot.subsystems;
+
+// import java.util.HashMap;
+import java.util.List;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatch;
+
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.I2C.Port;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class ControlPanelSubsystem extends SubsystemBase {
+
+    private VictorSPX cpMotor;
+    private DoubleSolenoid cpSolenoid;
+    private ColorSensorV3 colorSensor;
+    private DriverStation ds;
+    private ColorMatch colorMatcher;
+
+    private Color turnToColor = null;
+    private Color detectedColor = null;
+    private Color fmsColor = null;
+    private Color lastColor = null;
+
+    private List<String> colorNames;
+    private List<Color> _colors;
```

Comment:
why the underscore?

Reply:
Ah, that was because I am unoriginal about naming variables, so when I tried experimenting with using a LinkedHashMap instead of the two Lists, I called the LinkedHashMap "colors", and then just added an underscore to the original variable name because I didn't want to completely cut it out yet. If/When I figure out how to best use a LinkedHashMap, I'll take out those two Lists.